postgres/values.yaml

```diff
@@ -0,0 +1,66 @@
+---
+replicaCount: 1
+image: "postgres:9.6"
+
+## Specify postgres username/password
+##
+# postgresUser:
+# postgresPassword:
+# postgresDatabase:
+
+## Specify a imagePullPolicy
+## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+##
+# imagePullPolicy:
+
+## Specify imagePullSecrets
+## ref: https://kubernetes.io/docs/concepts/containers/images/#specifying-imagepullsecrets-on-a-pod
+##
+# imagePullSecrets: myregistrykey
+
+secret:
+  enabled: false
+
+  ## Use secret store to set postgres user password
+  ## ref: https://kubernetes.io/docs/concepts/configuration/secret/
+  ##
+  # name:
+  # key:
+
+
+## Persist data to a persitent volume
+persistence:
+  enabled: false
+
+  ## database data Persistent Volume Storage Class
+  ##
+  # storageClass:
+  # accessMode: ReadWriteOnce
+  # subPath:
+  # size: 8Gi
+
+  # annotations: {}
+
+## Configure resource requests and limits
+## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+##
+resources:
+  requests:
+    memory: 256Mi
+    cpu: 100m
```

Comment:
optionnal?

Reply:
I prefer keep an default, to ensure resource avaibility on the node. You can override this comportement by setting `resources` to `{}`.

Reply:
ok but we need to be consistant